osbuild/util/osrelease.py

```diff
@@ -33,7 +33,7 @@ def parse_files(*paths):
                 if line[0] == "#":
                     continue
                 key, value = line.split("=", 1)
-                osrelease[key] = value.strip('"')
+                osrelease[key] = value.strip('\'"')
```

Comment:
I think this is still not a correct parser; from [the doc](https://www.freedesktop.org/software/systemd/man/latest/os-release.html), particularly this bit:

> Shell special characters ("$", quotes, backslash, backtick) must be escaped with backslashes, following shell style.

I think it'd be correct to use [shlex.split](https://docs.python.org/3/library/shlex.html) and error if there's more than one value for example.

This would also handle quoting in both styles via `"` or `'`.

Reply:
Great! Thanks for checking it @cgwalters !!